lib/src/core/toastification_manager.dart

```diff
@@ -27,6 +27,8 @@ class ToastificationManager {
   /// if the list is empty, the overlay entry will be removed
   final List<ToastificationItem> _notifications = [];
 
+  Duration delay = const Duration(milliseconds: 10);
```

Comment:
Creating a class field as delay is not really make sense

When we are creating the overlay in line 60 and we will update this value to 300 milliseconds and we always use that value

Basicity the delay value in every situation is 300 milliseconds 
I'm going to test and see what are the side effects